module5/snippets/home-snippet.html

```diff
@@ -7,7 +7,7 @@
         <a href="#" onclick="$dc.loadMenuCategories();"><div id="menu-tile"><span>menu</span></div></a>
       </div>
       <div class="col-md-4 col-sm-6 col-xs-12">
-        <a href="#" onclick="$dc.loadMenuItems({{randomCategoryShortName}});">
+        <a href="#" onclick="$dc.loadMenuItems('{{randomCategoryShortName}}');">
```

Comment:
why did you make changes to this file? the homework instructions specifically say do not change this file.